### DISCUSSION TITLE
CARLA Docker .sh script issues

### DISCUSSION THREAD

**Discussion by adiancook** [carla-simulator/carla#5622] [Q&A]
Hi Everyone,

Ubuntu 18.04, latest carla version, Docker version 20.10.17, build 100c701.

I'm having an issue with this command:
docker build -t carla -f Carla.Dockerfile .
 
returning this error:
--2022-07-20 03:50:51--  https://www.autodesk.com/content/dam/autodesk/www/adn/fbx/2020-0-1/fbx202001_fbxsdk_linux.tar.gz
Resolving www.autodesk.com (www.autodesk.com)... 104.97.127.127
Connecting to www.autodesk.com (www.autodesk.com)|104.97.127.127|:443... connected.
HTTP request sent, awaiting response... 403 Forbidden
2022-07-20 03:50:51 ERROR 403: Forbidden.

I modified the BuildUtilsDocker.sh wget command to:
wget --user-agent="Mozilla" https://www.autodesk.com/ -c "${FBXSDK_URL}" -P "${CARLA_DOCKER_UTILS_FOLDER}"
but that only fixed my error for a single build. 

The next error I'm debugging is:
Cloning into '/home/carla/carla/Build/libosm2dr-source'...
Checking out files: 100% (78068/78068), done.
fatal: unable to access 'https://github.com/carla-simulator/sumo.git/': Failed to connect to github.com port 443: Connection timed out
Util/BuildTools/Linux.mk:152: recipe for target 'osm2odr' failed
make: *** [osm2odr] Error 128
The command '/bin/sh -c cd /home/carla/ &&   if [ -z ${GIT_BRANCH+x} ]; then git clone --depth 1 https://github.com/carla-simulator/carla.git;   else git clone --depth 1 --branch $GIT_BRANCH https://github.com/carla-simulator/carla.git; fi &&   cd /home/carla/carla &&   ./Update.sh &&   make CarlaUE4Editor &&   make PythonAPI &&   make build.utils &&   make package &&   rm -r /home/carla/carla/Dist' returned a non-zero code: 2

But now I'm stuck back on the autodesk issue. Any guidance here?

**Comment by HaechanMarkBong** [comment]
Did you have any luck with this?

**Reply by adiancook** [reply]
Nope, nothing still. Tried different networks, VPN's, git settings, and 2 Linux laptops. No luck yet!
